src/popup/router/pages/TipPage.vue

```diff
@@ -256,45 +241,71 @@ export default {
 };
 </script>
 
-<style lang="scss" scoped>
+<style lang="scss">
```

Comment:
Can you use `/deep/` instead of this to point what selector changes styles of child components? In general, I would suggest avoiding changing styles of nested components.